deployments/scheduler/scheduler.yaml

```diff
@@ -16,6 +16,15 @@ spec:
       containers:
         - name: yunikorn-scheduler-k8s
           image: yunikorn/yunikorn-scheduler-k8s:latest
+          env:
+            - name: logEncoding
+              value: console
+            - name: logLevel
+              value: '1'
```

Comment:
let's use `-1` for now, that is DEBUG level.

Reply:
Done